tuxemon/event/actions/variable_math.py

```diff
@@ -3,14 +3,23 @@
 from __future__ import annotations
 
 import logging
+from collections.abc import Callable, Mapping
 from dataclasses import dataclass
-from typing import Union, final
+from operator import add, floordiv, mul, sub
+from typing import Optional, final
 
 from tuxemon.event.eventaction import EventAction
 from tuxemon.tools import number_or_variable
 
 logger = logging.getLogger(__name__)
 
+ops_dict: Mapping[str, Callable[[float, float], float]] = {
```

Comment:
We should consider moving this dictionary to a central place, instead of defining it everywhere we need it.
Not necessary for this PR, just a thought I had.

Reply:
I agree, I wrote about it above, I would like to put in in tools.py